index.html

```diff
@@ -2449,8 +2449,8 @@ <h5>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">因注文宽度经常超过基文，通常为正文设定较大的字距以供给注文足够的横向伸展空间，避免基文间距频繁被过宽的注文撑大。</p>
           </li>
           <li id="id147">
-            <p its-locale-filter-list="en" lang="en">The phonetic annotations are all in lowercase. Sentence case is rare.</p>
-            <p its-locale-filter-list="zh-hans" lang="zh-hans" class="checkme">注文全小写，有时专名首字母大写。</p>
+            <p its-locale-filter-list="en" lang="en">The phonetic annotations are all in lowercase. Capitalizing proper nouns is rare.</p>
```

Comment:
There's a significant change to the meaning here in the English.  Is that reflected in the chinese?  I ask because i did just recently see pinyin that capitalised the initial letter in sentences, so that's a thing.

Reply:
I updated this paragraph because the original translation was incorrect. Wrt whether capitalizing proper nouns is commonly used, I'll let @ryukeikun answer that.

Reply:
@r12a In yesterday's [meeting](https://www.w3.org/2020/03/05-clreq-minutes.html#t02), Eric said that capitalizing proper nouns is not rare, so I fixed the Chinese and English text.